framework/src/main/java/org/checkerframework/framework/flow/CFAbstractValue.java

```diff
@@ -68,6 +68,12 @@ public abstract class CFAbstractValue<V extends CFAbstractValue<V>> implements A
   /** The annotations in this abstract value. */
   protected final AnnotationMirrorSet annotations;
 
+  /** The then store. */
```

Comment:
Please say more here.  The pull request doesn't seem to have any comments about the feature being implemented or the purpose of the code, and I think that will make it unnecessarily hard for readers of the code to understand the feature and its implementation.